apis/types/auth_config.go

```diff
@@ -7,6 +7,7 @@ package types
 
 import (
 	strfmt "github.com/go-openapi/strfmt"
+
```

Comment:
@houstar Hi. I found that there are considerable files which are added the invalid blank line in the import path.

In pouchContainer or dragonfly, we suggest we should obey the standard order of import path:

```
We suggest split the imports into three group with a blank:

the packages belong to go standard library
the packages belong to our project
the packages belong to Third-party
```
Please revise your PR. And I will help you review it.

Reply:
Please check your swagger's version, and we now use 0.17.1, see https://github.com/alibaba/pouch/blob/eb49d7c198f54c7ffcf95e8493f45d82b34863c3/hack/codegen/swagger.sh#L5.

If it is not the same version, please update swagger to 0.17.1. @houstar

Reply:
OK, wil update swagger to the specified version